src/serac/numerics/quadrature_data.hpp

```diff
@@ -139,6 +139,11 @@ class QuadratureData : public SyncableData, public QuadratureDataImpl<T, Quadrat
    */
   QuadratureData(mfem::Mesh& mesh, const int p, const bool alloc = true);
 
+// Turn off null dereference warnings for GCC
+// TODO add errors if the MFEM types return null pointers
```

Comment:
It looks like you already added an error below. Is this TODO referring to other cases?

Reply:
it's more a TODO to fix the underlying warning instead of turning it off. I'll clarify the comment.